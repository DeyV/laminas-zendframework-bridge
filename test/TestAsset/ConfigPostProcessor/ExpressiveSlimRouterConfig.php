<?php
namespace Acelaya\Expressive;

return array(
    'dependencies' => array(
        'factories' => array(
            'Zend\Expressive\Router\RouterInterface' => 'Acelaya\Expressive\Factory\SlimRouterFactory',
        ),
    ),
);
