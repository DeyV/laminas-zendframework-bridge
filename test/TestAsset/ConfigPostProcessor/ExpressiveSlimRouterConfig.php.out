<?php
namespace Acelaya\Expressive;

return array(
    'dependencies' => array(
        'factories' => array(
            'Mezzio\Router\RouterInterface' => 'Acelaya\Expressive\Factory\SlimRouterFactory',
        ),
    ),
);
